src/amo/components/UserProfile/index.js

```diff
@@ -118,6 +118,22 @@ export class UserProfileBase extends React.Component<Props> {
                   </a>
                 </Definition>
               ) : null}
+              {user && user.location ? (
+                <Definition
+                  className="UserProfile-location"
+                  term={i18n.gettext('Location')}
+                >
+                  {user.location}
+                </Definition>
+              ) : null}
+              {user && user.occupation ? (
+                <Definition
+                  className="UserProfile-occupation"
+                  term={i18n.gettext('Occupation')}
+                >
+                  {user.occupation}
+                </Definition>
+              ) : null}
```

Comment:
hey @willdurand this looks really good but i just had a couple of questions on some related topics:

do we want show the LoadingText in this scenario? I am not sure if it would be useful here since they are optional fields and by the time you figure if you are supposed to show i guess it's ready but I saw it on some of the other fields just double checking..

regarding styles - i am new to sketch so i am not 100% i am reading this right and I am not even sure that i am looking at the right mock  : ) (AMO - Desktop Master) but it *looked * like maybe the fonts were a little bigger:
![Alt text](https://monosnap.com/image/hcFfjYJVwrXa8sVupk1tQ0MmyEbtLP.png)

what do you think?

the following is def out of scope but noticed the following (should a new / separate issue be opened for this?):
![Alt text](https://monosnap.com/image/41o3uJoT4uDyEIumzgDbHvuf5RU1Pf.png)

Reply:
> do we want show the LoadingText in this scenario? I am not sure if it would be useful here since they are optional fields and by the time you figure if you are supposed to show i guess it's ready but I saw it on some of the other fields just double checking..

I did not show the `LoadingText` because those fields are optional and `homepage` (which is also an optional field) was already there, without a loading indicator...

> regarding styles - i am new to sketch so i am not 100% i am reading this right and I am not even sure that i am looking at the right mock : ) (AMO - Desktop Master) but it *looked * like maybe the fonts were supposed to be a little big bigger:

Yes, this is the right file and fonts are indeed different. The font sizes in the app and in the Sketch files are unfortunately different. I think that's because we still have to adjust a few things in reality and cannot take the Sketch values as is.

> the following is def out of scope but noticed the following (should a new / separate issue be opened for this?):

mmh yes we definitely have to open a new issue here, good catch!